src/components/Dropdown/Dropdown.tsx

```diff
@@ -203,9 +203,7 @@ function useFocusContextInstance(
         setFocusedIndex(index);
         setFocusedValue(items[index].value);
         setFocusedItemData(items[index].itemData);
-        if (focusedIndex === -1) {
-          setAlwaysSelectScreenReaderText(screenReaderText + ' ' + items[index].value);
-        }
+        setAlwaysSelectScreenReaderText(screenReaderText + ' ' + items[index].value);
```

Comment:
I think this would read the options count screen reader text even when arrowing onto dropdown options instead of just when the input text changes. when navigating with keyboard, we would only want the dropdown item's value to be announced